src/helpers/response-status.helper.test.ts

```diff
@@ -0,0 +1,174 @@
+import { HttpStatus } from '@nestjs/common';
+import { isAxiosError } from 'axios';
+
+import { messageCheck, statusCheck } from './response-status.helper';
+
+jest.mock('axios', () => ({
+  isAxiosError: jest.fn(),
+}));
+
+describe('statusCheck', () => {
+  beforeEach(() => {
+    (isAxiosError as unknown as jest.Mock).mockReturnValue(true);
+  });
+
+  it('should return true when isAxiosError is true and status matches', () => {
+    const error = {
+      response: {
+        status: HttpStatus.BAD_REQUEST,
+      },
+      name: 'Bad request',
+      message: 'Bad request',
+    } as Error;
```

Comment:
Should type be AxiosError from 'axios'?

Reply:
It should -- This should not be working without this.